packages/pkgsrc.rb

```diff
@@ -3,31 +3,19 @@
 class Pkgsrc < Package
   description 'pkgsrc is a framework for building third-party software on NetBSD and other UNIX-like systems'
   homepage 'https://github.com/NetBSD/pkgsrc'
-  version '2020Q1'
+  version '2020Q1-1'
   license '' # Can't find license
```

Comment:
I'm guessing it's BSD. :)

Reply:
However, I don't wanna misprovide anything. We do have a disclaimer but still. It might be a BSD-2 clause or BSD-3 clause